lib/IRGen/GenDistributed.cpp

```diff
@@ -693,7 +693,7 @@ void DistributedAccessor::emit() {
   }
 
   // Add all of the substitutions to the explosion
-  if (auto *genericEnvironment = Target->getGenericEnvironment()) {
+  if (Target->getGenericEnvironment()) {
```

Comment:
just check isGenericContext()

Reply:
Thanks!

Reply:
Hmm that's on a decl context and seems to be slightly wrong... but isGeneric() seems right~ 👍